src/mentors.json

```diff
@@ -6057,5 +6057,33 @@
         "id": "mannuelferreira.com"
       }
     ]
+  },
+  {
+    "id": "kevin.kipp@gmail.com",
+    "name": "Kevin Kipp",
+    "avatar": "https://pbs.twimg.com/profile_images/931552869915926528/sBpw-pNh_400x400.jpg",
+    "title": "Systems Engineer",
+    "description": "Front End Specialist & Functional Programming Enthusiast",
+    "country": "US",
+    "tags": [
+      "javascript",
+      "typescript",
+      "react",
+      "functional programming"
```

Comment:
The tag is too long. We had to limit the length so it will fit the ui. You can add that to the description.

Reply:
You already have it in the description :)